go/oasis-test-runner/oasis/keymanager.go

```diff
@@ -18,6 +20,125 @@ const (
 	keymanagerIdentitySeedTemplate = "ekiden node keymanager %d"
 )
 
+// KeymanagerPolicy is an Oasis key manager policy document.
+type KeymanagerPolicy struct {
+	net *Network
+	dir *env.Dir
+
+	statusArgs []string
+
+	runtime *Runtime
+	serial  int
+}
+
+// KeymanagerPolicyCfg is an Oasis key manager policy document configuration.
+type KeymanagerPolicyCfg struct {
+	Runtime *Runtime
+	Serial  int
+}
+
+func (pol *KeymanagerPolicy) provisionStatusArgs() []string {
+	return pol.statusArgs
+}
+
+func (pol *KeymanagerPolicy) provision() error {
+	if pol.runtime.teeHardware == node.TEEHardwareInvalid {
+		// No policy document.
+		pol.statusArgs = append(pol.statusArgs, "--keymanager.policy.file", "")
+	} else {
+		// Policy signed with test keys.
+		policyPath := filepath.Join(pol.dir.String(), kmPolicyFile)
+		policyArgs := []string{
+			"keymanager", "init_policy",
+			"--debug.dont_blame_oasis",
+			"--keymanager.policy.file", policyPath,
```

Comment:
"--"+`CfgPolicyFile` ?

Reply:
Likewise, was like that originally, but sure.